specification/cognitiveservices/data-plane/QnAMaker/stable/v4.0/QnAMaker.json

```diff
@@ -828,18 +828,6 @@
         }
       }
     },
-    "KnowledgebaseEnvironment": {
```

Comment:
Is this removal considered as a breaking change? Breaking changes should not happen in a stable version

Reply:
My understanding is that this is only used once for this file's redundant schema reference. I would assume that removing this has no breaking impact on the schema. Are you aware of a reason this would be a breaking change?

Reply:
So you are saying that removal of this schema will not cause a breaking change. If so, I am ok with this, and will approve this as soon as azure api board responds

Reply:
Is there a change to the REST API that motivated this change?

Reply:
Per offline conversation, this is not a change to the public REST API. Thus, no need for the Azure API review board to review.

Reply:
Hi @alargeasteroid after a check on SDK, this removal will cause a breaking change on go SDK (we have a tool for that).